samza-kv-rocksdb/src/main/scala/org/apache/samza/storage/kv/RocksDbKeyValueStore.scala

```diff
@@ -90,6 +90,22 @@ object RocksDbKeyValueStore extends Logging {
         }
       }
   }
+
+  def openTelemeterizedDB(dir: File, options: Options, storeConfig: Config, isLoggedStore: Boolean, storeName: String, metrics: KeyValueStoreMetrics): RocksDB = {
+    val rocksDb = openDB(dir, options, storeConfig, isLoggedStore, storeName)
+
+    if (storeConfig.containsKey("rocksdb.telemetry.list"))
```

Comment:
Can move this inside openDB() since this should be generally useful.

Reply:
Can't really do that as it requires a `RocksDB` instance (which `openDB` returns). Is it ok to leave as in in this case?

Reply:
We can do something like this in RocksDbKeyValueStore:71
```
try {
  var rocksDb = 
      if (ttl) {
          ....
      } else {
          ....
      }

  if (storeConfig.containsKey("rocksdb.telemetry.list")) {
      .... () -> metrics.newGauge(rocksDb.getProperty())
  }

  rocksDb
} catch {
}
```

But whichever version you prefer is fine.

Reply:
Ah ok i like your approach - will do that 👍